bundle/config/artifact.go

```diff
@@ -19,6 +19,7 @@ func (artifacts Artifacts) ConfigureConfigFilePath() {
 type ArtifactType string
 
 const ArtifactPythonWheel ArtifactType = `whl`
+const ArtifactPythonRequirementsFile ArtifactType = `requirements`
```

Comment:
requirements is not an artifact so we don't need to a new type but instead we should treat it as normal file